exportAllMembers.py

```diff
@@ -0,0 +1,81 @@
+from people import people
+#from groups import groups
+#from maps import maps
+#import config
+#import time
+#from geopy import distance
```

Comment:
do me a favor and remove the comments out of here. Looks like a copy of the original file, so we don't need to save it here